Makefile

```diff
@@ -1,15 +1,19 @@
 GOVERSION := $(shell go version | cut -d ' ' -f 3 | cut -d '.' -f 2)
 
-.PHONY: build check fmt lint test test-race vet test-cover-html help install proto
+.PHONY: build check fmt lint test test-race vet test-cover-html help install proto ui
 .DEFAULT_GOAL := build
 PROTON_COMMIT := "056db0eef064bc5f82343e6a5a312567a39d9ec6"
 
+ui:
```

Comment:
We should also ignore `ui` folder from various `test` commands

Reply:
just added into make test, rest of them are using specific folder to run test